examples/plot_9_plotly.py

```diff
@@ -0,0 +1,64 @@
+"""
+========================================
+Example with the plotly graphing library
+========================================
+
+sphinx-gallery supports examples made with the
+`plotly library <https://plotly.com/python/>`_. sphinx-gallery captures the
+``_repr_html_`` of plotly figure objects. Therefore you should return a figure
+object to display the figure.
+
+In order to use plotly, the ``conf.py`` of the project should include the
+following lines to select the appropriate plotly renderer::
+
+    import plotly.io as pio
+    pio.renderers.default = 'sphinx_gallery'
+
+**Optional**: the sphinx_gallery renderer of plotly will not generate png
```

Comment:
```suggestion
**Optional**: the ``sphinx_gallery`` renderer of plotly will not generate png
```
(to be consistent with formatting of sphinx_gallery_png below)